src/main/webapp/scripts/popular.js

```diff
@@ -0,0 +1,33 @@
+function loadPopular() {
+  loadApi(function(){
+    getPopular().then((response) => {
+      for(item of response.items) {
+        // Ready in a later PR: display the item nicely on the page
+        console.log(item);
+      }
+    });
+  });
+}
+
+function getPopular() {
+  // Ready in a later PR: let user choose country, currently defaulting to US
+  return gapi.client.youtube.videos.list(getPopularRequest('US'))
+  .then(function(response) {
+    return response.result;
+  })
+  .catch((err) => {
+    alert('Oops! We were not able to get the videos from Youtube');
+    console.log('An error occurred when making API list request:', err);
+  })
+}
+
+function getPopularRequest(regionCode) {
+  return {
+    'part': [
+      'snippet,contentDetails,statistics'
```

Comment:
I am not sure how we want to show the video. If we wish to do something with the id (like embed the video as we discussed before), maybe only the 'snippet' is needed?

Reply:
I need all 3 right now but if that changes I'll change this. (This is the list of recommended videos so its not the part involving embedding)